_posts/2022-05-28-classifieds-crawler.md

```diff
@@ -5,4 +5,18 @@ title:  "Classifieds Crawler"
 #### Team:  
 * Idea: Nikita
 * Project Manager: Stas
-* Project Developer: Julia
\ No newline at end of file
+* Project Developer: Julia
+
+### Description:  
+collecting some ads by conditions. After that saving them and notify when
+we have something new
+**Challenges:**  
+* Find and collect some ads (cheap or rear lots) on the sites (olx etc.) 
+* Collecting with conditions and filters
+* Saving the results
+* spliting new results and existing results
+* notifying (for instance, in telegram) when we get something new
+
+
+**Main challenges:**  
+* This type of sites doesn't "like" crawling
```

Comment:
Please add packages required for implementation of the project. E.g. requests, web spider, beautiful soup etc.